.github/workflows/ci.yml

```diff
@@ -232,13 +233,13 @@ jobs:
           command: pytest --tag ${{ matrix.tags }} --kwdof
           attempt_limit: 3
 
-      - name: Upload logs on failure
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          name: logs-${{ matrix.tags }}-${{ matrix.profile }}-${{ matrix.nxf_version }}
-          path: |
-            /tmp/pytest_workflow_*/*/.nextflow.log
-            /tmp/pytest_workflow_*/*/log.out
-            /tmp/pytest_workflow_*/*/log.err
-            /tmp/pytest_workflow_*/*/work
+      # - name: Upload logs on failure
+      #   if: failure()
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     name: logs-${{ matrix.tags }}-${{ matrix.profile }}-${{ matrix.nextflow_version }}
+      #     path: |
+      #       /tmp/pytest_workflow_*/*/.nextflow.log
+      #       /tmp/pytest_workflow_*/*/log.out
+      #       /tmp/pytest_workflow_*/*/log.err
+      #       /tmp/pytest_workflow_*/*/work
```

Comment:
why commenting that part?

Reply:
It didnt work but I just saw how you do it in your PR and I have used it :)

Reply:
Will be changed today

Reply:
Also you do seem to have looooots of test.
On Sarek, we do only test `-profile test` on both version of nextflow (minimal + latest) (but docker, singularity and conda).
But we do have all other tests on just the minimal version.
Not sure what we should recommend or not there actually.
But ~200 seems a bit too much

Reply:
nah I like the tests - its only me developing the pipeline so I really need lots of coverage to ensure all the possibilities are covered